app/experimenter/templates/experiments/section_analysis.html

```diff
@@ -7,4 +7,25 @@
 {% block section_content %}
   <p><strong>Data Science Owner: {{ experiment.analysis_owner }}</strong></p>
   {{ experiment.analysis|urlize|linebreaks }}
+
+  {{ experiment.SURVEY_REQUIRED_LABEL }}
+  <strong class="{% if experiment.survey %}text-danger{% endif %}">
+    {{ experiment.survey|yesno:"Yes,No" }}
+  </strong>
+
+  {% if experiment.survey %}
+  <br>
+    <strong>Survey URLs</strong>
+    <p>
+      {{ experiment.survey_urls }}
+    </p>
+
+    <strong>Survey Launch Instructions</strong>
+    <p>
+      {{ experiment.survey_instructions }}
+    </p>
+  {% endif %}
+
+
+
```

Comment:
Per my discussion in the main part of this PR... in this section overview, we could add a title inside the box for surveys... but we might only want to show this title if there is a survey..? Because if we have this title even if there isn't a survey.. there would be a big title with only one line below it. Maybe that's fine? I dunno, this part of the UI probably needs more.

Reply:
I think this is fine, and Shell seemed OK with it too!